src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -130,6 +138,17 @@ && isNotBlank(appellant.getAddress().getPostcode())) {
         return preSubmitCallbackResponse;
     }
 
+    private void validateBenefitIssueCode(SscsCaseData caseData,
+                                          PreSubmitCallbackResponse<SscsCaseData> response) {
+        boolean secondDoctor = isNotBlank(caseData.getSscsIndustrialInjuriesData().getSecondPanelDoctorSpecialism());
```

Comment:
Minor - you could rename secondDoctor to be more explicitly a boolean, something like isSecondDoctorPresent. More verbose but more descriptive.

Reply:
done